### DISCUSSION TITLE
How to use the simulator?

### DISCUSSION THREAD

**Discussion by heygarrett** [kkharji/xbase#152] [Q&A]
Hi! I'm a long time (Neo)vim user, but I'm new to app development.

The readme mentions:  
> Run your products on simulators relative to your target's platform. (+ watch build and ran on change)

but I can't find details about how to do this. Is it something I need to initiate from my shell, or is there a built-in Lua function/Vim command to facilitate it?

**Comment by kkharji** [comment]
@heygarrett , it is a builtin function, you can access it by the telescope picker for now. If you are using default configuration. You can access the function through a shortcut `<leader>r` or `<leader>s`.  To access all pickers you can run `<leader>ef`

Let me know if this solves it for you and I hope you get up to speed with app development soon
